backend/src/main/java/me/quizzl/backend/models/Grade.java

```diff
@@ -0,0 +1,54 @@
+package me.quizzl.backend.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+
+@Entity
+public class Grade {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="id", updatable = false, nullable = false)
+    protected Long id;
+
+    @OneToOne
+    @JoinColumn(name="submission_id", nullable = false)
+    protected Submission submission;
+
+    @Column(name = "number_correct")
+    protected int numCorrect = 0;
```

Comment:
just a thought: we could probably just store whether or not each answer is correct inside of the `Answer` class, under a boolean. if we do that, we could just calculate the grade by tallying up the correct answers over the total answers, which would mean that we could drop this `Grade` entity entirely